dist/ExtUtils-ParseXS/lib/ExtUtils/ParseXS.pm

```diff
@@ -690,10 +690,17 @@ EOF
         do_push     => undef,
       } ) for grep $self->{in_out}->{$_} =~ /OUT$/, sort keys %{ $self->{in_out} };
 
-      my $prepush_done;
+      my $c = @{ $outlist_ref };
```

Comment:
Surely this can have a better name than `$c`

Reply:
fixed